### ISSUE TITLE
Allows users to send emails for their own email account

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/backend#428]
Background discussion in #412

**Comment by markmacgillivray** [comment]
I'm a bit confused by this - on the last issue I said we should not push people towards using their own email accounts to contact authors because it leaves us with no way to track what is going on - therefore, the app we run serves no purpose in that context.

But this does NOT mean that users cannot email authors from their own email accounts - of course they can. Anyone in the world can email anyone else from their own account.

The point is that if anyone went to the URL of an article they wanted access to, and they found the email address of an author, and they chose to email that author to ask for access... that is all something that anyone can already do, and there is no app needed to achieve that. So there is no point in oabutton providing such functionality.

That is why I think the oabutton could provide on the request page an option for users to send an email to the author, and we cc the user into the email we send. That way, the oabutton app is doing something for them - publicising the request, potentially gathering community support, other potential avenues for the request to be fulfilled, etc, and we track the progress.

So what else is there left to actually answer in this issue itself as it stands? We don't (and can't) do anything right now to stop our users or anyone else from sending emails from their own account, so what is it we are trying to decide here?

**Comment by chealsye** [comment]
I agree with Mark. Users can email on their own already and the Button doesn't need to fulfill it, but the worst part is we wouldn't know if the author fulfills the request. The tricky thing is thought that authors typically can legally send an individual the publisher's PDF, which they couldn't through the Button/IR. 

There will be many cases were an article is embargoed for 12 months and we can't do anything to help the user until then. It may be worth providing a template email for a user to send to an author if they wanted to request a copy though... for an advocacy standpoint and helping immediately, I like that.

**Comment by markmacgillivray** [comment]
Can probably be closed.
